Arrowgene.Ddon.GameServer/Quests/Quest.cs

```diff
@@ -385,7 +385,7 @@ public virtual CDataTimeGainQuestList ToCDataTimeGainQuestList(uint step)
                 {
                     ItemId = reward.ItemId,
                     Num = reward.Num,
-                    Type = 12 // What does this type mean?
+                    Type = 12
```

Comment:
have we figured it out?

Reply:
I think it might be one of these, but still not sure.

```c
  QUEST_REWARD_TYPE_UNKNOWN = 0x0,                                                                                                          
  QUEST_REWARD_TYPE_FIXED = 0x1,
  QUEST_REWARD_TYPE_SELECT = 0x2,                                                                                                           
  QUEST_REWARD_TYPE_UNDISCOVERY = 0x3,
  QUEST_REWARD_TYPE_RANDOM = 0x4,
  QUEST_REWARD_TYPE_REPEAT = 0x5,
  QUEST_REWARD_TYPE_SWITCH = 0x6,
  QUEST_REWARD_TYPE_BORDER = 0x7,
  QUEST_REWARD_TYPE_RANKING = 0x8,
  QUEST_REWARD_TYPE_CHARGE = 0x9,
  QUEST_REWARD_TYPE_REGION_BREAK = 0xA,
  QUEST_REWARD_TYPE_FIXED_FIRST = 0xB,
  QUEST_REWARD_TYPE_FIXED_SECOND = 0xC,
  QUEST_REWARD_TYPE_FIXED_MEMBER_FIRST = 0xD,
  QUEST_REWARD_TYPE_PROGRESS_BONUS = 0xE,
  QUEST_REWARD_TYPE_NUM = 0xF,
```